.github/workflows/publish-cdn.yml

```
@@ -0,0 +1,50 @@
+name: Publish CDN artifacts (versioned) to S3
+run-name: Publish CDN artifacts - versioned by @${{ github.actor }}
+
+on:
+  workflow_dispatch:
```

```suggestion

on:
  push:
    branches:
      - main
```

@Palke Yes, this should probably be set to run on pushes to `main`. Otherwise artifacts will deploy to the CDN for every PR (unless there's a workflow that will actually use them?). I'm not sure why the workflow for Labs runs during dispatch, I'm not too familiar with that project.

---

@kennylam do you know if there is any way to roll back changes? If we push to CDN build from this repository and something goes wrong, can we recover easily, any idea?

---

My concern is mostly because of the setup is completely rebuild and also the script `load-non-latin-plex.js` is actually generated now by webpack and not gulp. There has been update to the latest dotcom packages and the generated file is +- twice as big as the original one on CDN. 

This is why I was wondering if there is a way to test this new setup - or rollback to what CDN currently is using.

I did compared generated CSS files and their names to currently existing structure, but would like to have some safety plan in place, as rollback to current version. Any thoughts on that?

---

We can (and should) test it once this gets merged. That shouldn't be difficult to do.